### PR TITLE
subset of units improved to reduce computation

### PR DESCRIPTION
**Pull request by colehurwitz**
Computation of metrics performed only on specified units. Identical values as doing all the units, but should be more efficient!

### CODE REVIEW

**Comment by colehurwitz** [comment]
Needs spikemetrics PR to work

File: setup.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
         'numpy',
         'spikeextractors',
         'spikesorters',
-        'spikemetrics >= 0.1.5',
+        'spikemetrics >= 0.1.6',
```
Comment (alejoe91) [inline setup.py]: isn't it 0.2.0?

Comment (colehurwitz) [reply setup.py]: Updated in latest commits

**Comment by colehurwitz** [comment]
merging

**Pull request merged**